README.md

```diff
@@ -1,12 +1,56 @@
+<p align="center">
+  <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-320.png" />
+</p>
+
+---
+
 # Android Common Extensions
 
 This repository provides a collection of common Android extensions.
 
+[![](https://jitpack.io/v/nimblehq/android-common-ktx.svg)](https://jitpack.io/#nimblehq/android-common-ktx)
+
 ## Usage
 
-Clone the repository
+1. Add JitPack to your project build.gradle
```

Comment:
@hoangnguyen92dn Suggestion:

```
Add JitPack to your project `build.gradle` or `settings.gradle`
```

Reply:
@doannimble It should be `or` or `and`? 🤔